EvidenceApi.Tests/V1/UseCase/CreateDocumentSubmissionUseCaseTests.cs

```diff
@@ -15,20 +13,16 @@
 using NUnit.Framework;
 using System.Threading.Tasks;
 using EvidenceApi.V1.Domain.Enums;
-using Microsoft.Extensions.Logging;
 
 namespace EvidenceApi.Tests.V1.UseCase
 {
     public class CreateDocumentSubmissionUseCaseTests
     {
         private CreateDocumentSubmissionUseCase _classUnderTest;
         private Mock<IEvidenceGateway> _evidenceGateway = new Mock<IEvidenceGateway>();
-        private Mock<IResidentsGateway> _residentsGateway = new Mock<IResidentsGateway>();
```

Comment:
This file was amended because `CreateDocumentSubmissionUseCase` no longer sends an email to the officer -- it's been moved to `SendNotificationUploadConfirmationToResidentAndStaff`